2020-pip/RFP.md

```diff
@@ -0,0 +1,278 @@
+
+Work on the pip resolver by writing test cases/testing plan, testing maintainers' refactoring and new features, investigating and fixing bugs found in beta testing, and working with PSF on test infrastructure
+
+March:
+- Work on pip resolver; writing test cases and starting a testing plan, testing maintainers' refactoring and new features, and working with PSF on test infrastructure: 140 hours (4 fulltime weeks of 35 hrs each)
+
+April:
+ - Work on pip resolver; writing test cases and improving a testing plan, testing maintainers' refactoring and new features, and working with PSF on test infrastructure: 140 hours (4 fulltime weeks of 35 hrs each)
+   - (Part of this work to be done during [PyCon North America](https://us.pycon.org/2020/), one week, lodging and travel covered)
+
+May:
+ - Investigate and fix bugs found in beta testing, and improve and use testing plan: 40 hours (probably 4 weeks at 10 hours/week)
+   - We aim in this month to get pip's resolver feature prepared for release in pip 20.2 in July. (Per the quarterly release cadence for pip, unforeseen difficulties may delay till 20.3 in the next quarter.)
+
+#### June-December 2020 (Maintenance and Sustainability, Phase III)
+Once we have finished rolling out the resolver, the contractor will:
+- Keep up with the pip code and issue review queue for 10 hours per week, thus releasing a key bottleneck and enabling existing maintainers to make progress on key architectural features
+- Help new contributors develop into continuing contributors
+- Help existing contributors grow into co-maintainers
+
+
+June:
+- Review and respond to code contributions and new issues: 40 hours (probably 4 weeks at 10 hours/week)
+
+July:
+- _vacation, unpaid_
+
+August:
+- Review and respond to code contributions and new issues: 40 hours (probably 4 weeks at 10 hours/week)
+
+September:
+- Review and respond to code contributions and new issues: 40 hours (probably 4 weeks at 10 hours/week)
+
+October:
+- Review and respond to code contributions and new issues, and possibly participate in user experience training: 40 hours (probably 4 weeks at 10 hours/week)
+
+November:
+- Review and respond to code contributions and new issues: 30 hours (probably 3 weeks at 10 hours/week)
+
+December:
+- Review and respond to code contributions and new issues: 20 hours (probably 2 weeks at 10 hours/week)
+
+## Expectations and Requirements
+
+### Project Timeline
+
+This project is intended to be completed over three phases, totalling a little over one year, starting mid-December 2019 or early January 2020.
+
+Role 1 and Role 2 have different timelines; see those roles for details.
+
+### Technology
+
+The [pip codebase](https://github.com/pypa/pip) is licensed under the [MIT License](https://github.com/pypa/pip/blob/master/LICENSE.txt). All work submitted or dependencies added must be compliant with this license.
+
+The backend codebase is in Python. The pip documentation is built using reStructuredText and Sphinx.
```

Comment:
Which backend does this refer to?

Reply:
Ah right; sorry, "backend" here is extraneous and I will remove it.